src/psyclone/tests/psyir/frontend/fparser2_literals_test.py

```diff
@@ -109,6 +167,8 @@ def test_handling_literal_precision_1(value, dprecision, intrinsic):
     assert literal.datatype.intrinsic == intrinsic
     if intrinsic == ScalarType.Intrinsic.BOOLEAN:
         assert ".{0}.".format(literal.value) == value.lower()
+    elif intrinsic == ScalarType.Intrinsic.CHARACTER:
```

Comment:
As you don't use lower() here, perhaps make the test string mixed case to check that there is no unexpected lowering going on in the translation.

Reply:
Good idea, done.